src/GraphQL.MicrosoftDI/ScopedAttribute.cs

```diff
@@ -0,0 +1,21 @@
+using GraphQL.MicrosoftDI;
+using GraphQL.Types;
+
+namespace GraphQL
+{
+    /// <summary>
+    /// Creates a dedicated service scope during the field resolver's execution.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
```

Comment:
AllowMultiple = false

Reply:
https://docs.microsoft.com/en-us/dotnet/api/system.attributeusageattribute.allowmultiple?view=net-6.0

> true if more than one instance is allowed to be specified; otherwise, false. The default is false.

Reply:
ok